.github/workflows/pull_request.yaml

```diff
@@ -46,6 +46,20 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python }}
+      - name: Install yq
+        run: |
+          sudo add-apt-repository --yes ppa:rmescandon/yq
+          sudo apt update
+          sudo apt install yq -y
+      - name: Enable/Disable pulp-rpm tests
+        env:
+          DISABLE_PULP_RPM_TEST: ${{ secrets.DISABLE_PULP_RPM_TEST }}
+        run: |
+          if [[ "$DISABLE_PULP_RPM_TEST" == 'true' ]]; then
+            find ./molecule/*/group_vars/all -exec sh -c "yq d -i {} pulp_install_plugins.pulp-rpm" \;
+            find ./molecule/*/group_vars/all -exec sh -c "echo; echo {}; cat {}" \;
+          fi
+        shell: bash
```

Comment:
@pulp/ansible-installer I came up with this idea, we can disable pulp-rpm test by setting a secret for this repo

Reply:
Like an environment variable that if present would cause it to skip?

Reply:
exactly!

Reply:
That sounds great! Let's document it in the docs somehow for developers please too.

Reply:
so we can avoid:
https://github.com/pulp/pulp_installer/pull/321#discussion_r438310372
without touching the code

Reply:
@bmbouter only maintainers can change the repo environment variables, so I'm not sure where to document it

Reply:
I think in the part of the docs that talk about tests. Say for example someone went to run the tests locally and they wanted to know how to set the environment var locally. I don't see that section in the docs so I propose make a new one.

Reply:
the environment variable trick is only used on the CI

Reply:
Oh I see. I was imagining a skip-test on the test itself, but you're saying (I think) that this disables part of the matrix. If it's that second one then disregard my suggestions to document because it's only in the CI like I'm saying.

Reply:
yes, it is the second case. The matrix will install pulp-file and pulp-rpm regularly, but due release, we can remove pulp-rpm from the matrix (sometimes pulp-rpm is released couple days later than pulpcore/pulp-file which breaks the CI)